### ISSUE TITLE
Manifest merger fails with error.

### ISSUE THREAD

**Issue by cordel** [karczews/RxBroadcastReceiver#6]
Hey. My app's manifest contains following lines: 
   
-  `android:allowBackup="false"`
-   `android:supportsRtl="false"`

And it can't compile because of these issues: 
![image](https://user-images.githubusercontent.com/4090942/36665381-7514a39e-1af8-11e8-9665-87589ca44223.png)
![image](https://user-images.githubusercontent.com/4090942/36665393-7ea2b54a-1af8-11e8-8537-a352830d6a8d.png)

Now I use `tools:replace="android:allowBackup, android:supportsRtl"` as a quick workaround but as I understand that's not the desired behavior.

Could you please fix?

**Comment by ajoz** [comment]
Hi,
Could you describe me the configuration of your project. Is it a single app project? Or maybe several library projects + the app. I have a vague idea how to fix the issue you are having but I would like to reproduce it first and create some testing playground for it.

**Comment by cordel** [comment]
Okay. Reproduction steps:
Create new project
Add this dependency `implementation 'com.github.karczews:rx2-broadcast-receiver:1.0.4'`
Add next line to the manifest `android:supportsRtl="false" android:allowBackup="false"`
Start sync
Get the error

**Comment by ajoz** [comment]
Thank you. Like I said I have a vague idea how to solve it, but I didn't want to send changes blindly. I will take a look at this today and put all relevant information in the ticket.

**Comment by ajoz** [comment]
Hi,

I've checked your issue and it is already solved with our current `1.0.5-SNAPSHOT` build, please use it instead. It has a corrected manifest and I couldn't reproduce the issue with the manifest merger with it.

To use it please add `oss.sonatype.org` repositories to your dependencies. For example in the top level project `build.gradle`:

```gradle
allprojects {
    repositories {
        google()
        jcenter()
        maven {
            url "https://oss.sonatype.org/content/repositories/snapshots"
        }
    }
}
```

Then you can add to your `app` dependecies:

```gradle
implementation 'com.github.karczews:rx2-broadcast-receiver:1.0.5-SNAPSHOT'
```

So for now please use `1.0.5-SNAPSHOT` version, we will make a release to maven central in the comming week.

Good luck!

**Comment by cordel** [comment]
Awesome thank you!
